orttraining/tools/ci_test/run_batch_size_test.py

```diff
@@ -13,6 +13,7 @@ def parse_args():
     parser = argparse.ArgumentParser(description="Runs a BERT batch size test.")
     parser.add_argument("--binary_dir", required=True, help="Path to the ORT binary directory.")
     parser.add_argument("--model_root", required=True, help="Path to the model root directory.")
+    parser.add_argument("--gpu_sku", default='V100_16G', required=False, help="GPU model (e.g. V100, MI100).")
```

Comment:
maybe use [choices](https://docs.python.org/3/library/argparse.html#choices) here. also, might be good to have consistent values (e.g., V100_16G vs. V100) across the scripts

Reply:
Yeah, agreed.